core-web/src/main/java/org/silverpeas/core/web/look/DefaultLayoutConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package org.silverpeas.core.web.look;
+
+import org.silverpeas.core.util.SettingBundle;
+
+/**
+ * @author Nicolas Eysseric
+ */
+public class DefaultLayoutConfiguration implements LayoutConfiguration {
+
+  private SettingBundle settings;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "settings" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)